e2e_samples/mdw_governance/README.md

```diff
@@ -96,14 +99,14 @@ Each environment has an identical set of resources
    6. With administrative privileges on the subscription, run `az provider register --namespace 'Microsoft.Purview'` to register Purview provider. Alternatively you can use the Azure Portal to search for and select Subscriptions, select the subscription you want to use, and then select Resource providers. Search and register Microsoft.Purview provider.
    7. Create a new repository in your Azure DevOps project and copy the contents of the `e2e_samples\mdw-governance` folder to the root of this repository.
-   8. Update values of the variables to match the desired configuration by updating the `dev-environment.yml`, `test-environment.yml` and `prod-environment.yml` files to match your environment. Update at least:
-        - **resourceGroup** - this is the name of the Resource Group where the resources will be deployed
-        - **resourcePrefix** - this prefix will be appended to all resource names to make them unique (notice: lower-case letters only, no special characters allowed
-        - **location** - location where resources should be deployed
-        - **locationFormatted** - full name of location where resources should be deployed
-        - **purviewAdmins** - an [objectID](https://docs.microsoft.com/en-us/azure/marketplace/find-tenant-object-id) of the user/user group to be assigned admin rights to Purview (e.g. your Azure AD objectID). To find the object id of the current logged in user in `az cli`, run `az ad signed-in-user show --output json | jq -r '.objectId'`
-        - **azureResourceManagerConnection** - name of the service connection (by default this is derived from resource group name e.g. "SC-My-Resource-Group")
-   9. Create a new pipeline from existing yml in Azure DevOps by selecting the repository and importing the `create-infrastructure.yml` YAML file (see [this post](https://stackoverflow.com/a/59067271) to learn how).
+   8. Create a new pipeline from existing yml in Azure DevOps by selecting the repository and importing the `create-infrastructure.yml` YAML file (see [this post](https://stackoverflow.com/a/59067271) to learn how).
+   9. Make sure the values of the variables match the desired configuration and run the pipeline to deploy the core infrastructure. Update the dev-environment.yml, test-environment.yml and prod-environment.yml files to match your environment. Update at least:
+        - resourceGroup - this is the name of the Resource Group where the resources will be deployed
+        - resourcePrefix - this prefix will be appended to all resource names to make them unique (notice: lower-case letters only, no special characters allowed
+        - location - location where resources should be deployed
+        - locationFormatted - full name of location where resources should be deployed
+        - purviewAdmins - an [objectID](https://docs.microsoft.com/en-us/azure/marketplace/find-tenant-object-id) of the user/user group to be assigned admin rights to Purview (e.g. your Azure AD objectID)
```

Comment:
how about the new parameters needed for the pyspark script (column name, storage folders), maybe add a short explanation on what is done in the script?